email/finalizer.go

```diff
@@ -47,6 +47,7 @@ func NewFinalizer(ctx context.Context, database *database.AbuseScannerDB, emailC
 		staticEmailCredentials: emailCredentials,
 		staticLogger:           logger.WithField("module", "Finalizer"),
 		staticMailbox:          mailbox,
+		staticServerDomain:     serverDomain,
```

Comment:
decided to take this along, server domain was never assigned, has no real repercussions aside from some database metadata